src/App.jsx

```diff
@@ -105,6 +113,7 @@ function App() {
         </div>
       </div>
       <Footer />
+      <div className="absolute block w-full h-20 bg-gradient-to-b from-white -z-10-"></div>
```

Comment:
```suggestion
      <div className="absolute w-full h-20 bg-gradient-to-b from-white -z-10"></div>
```

- `<div>` 태그이므로 block은 불필요한 것 같습니다.
- -z-10-은 뒤에 -를 빼야할 것 같습니다.